montepy/input_parser/syntax_node.py

```diff
@@ -1728,6 +1832,13 @@ def end_padding(self, padding):
             )
         self._end_pad = padding
 
+    @property
+    def type(self):
+        """
+        TODO
+        """
+        return self._type
```

Comment:
What remains to be done?

Reply:
I still need to finish all the doc strings.